### ISSUE TITLE
Software is not compatible with high resolutions with high DPI

### ISSUE THREAD

**Issue by fredddii** [radareorg/cutter#999]
**Environment information**
* Operating System: Windows 10
* Cutter version: 1.7.2
* File format:

**Describe the bug**
Running 3200x1800 resolution. Windows is at the recommended scaling of 250%. Some elements of the software does not scale. The majority of it does just fine.

**To Reproduce**
1. Buy a high resolution screen or a device with one built in.
2. Run the software.

**Expected behavior**
For things to scale appropriately.

**Screenshots**
![cutter-high-dpi](https://user-images.githubusercontent.com/15217150/49964785-a14a4880-ff1b-11e8-9cce-ae8449df43bb.png)

**Additional context**
No.

**Comment by pelijah** [comment]
Related: #675

**Comment by ITAYC0HEN** [comment]
Ref: http://doc.qt.io/qt-5/highdpi.html

**Comment by ITAYC0HEN** [comment]
@thestr4ng3r do you have any idea about it?

**Comment by thestr4ng3r** [comment]
Not really, I would have to test myself. But right now I don't have any Windows dev environment set up.

**Comment by lillypad** [comment]
Running same DPI to fix this for just your main display do the following:

```batch
cmd.exe /c "set QT_AUTO_SCREEN_SCALE_FACTOR=1 & cutter.exe"
```

Currently If you take any other windows and move them to lower DPI monitors the windows look okay but the font will be either too small or too large.

Based off this documentation it should be something that can be looked at fixing in the future, I'm interested in helping out if I get some time.

__References:__
https://doc.qt.io/qt-5/highdpi.html

**Comment by ITAYC0HEN** [comment]
@lillypad thanks for the insights! Can we have your help with handling these issues with DPI? It would be great :)

**Comment by lillypad** [comment]
Yeah I have enough time to make a patch for a minor release to make it a little better.

**Comment by ITAYC0HEN** [comment]
Thank you very much :)

**Comment by ITAYC0HEN** [comment]
Hey @lillypad, any chance you took a look on this issue? :)

**Comment by karliss** [comment]
I started setting up build environments for testing  on different os and graphic environments and started considering to get a monitor.

Those environment varibles are somewhat temporary workarounds that can prevent UI from being terribly broken. We should use them if necessary until proper support is implemented.

**Comment by karliss** [comment]
Here are my work in progress notes for systematic approach to this https://gist.github.com/karliss/3e2da4c372ea9c49fab8652bfae0c7b8
